src/collision_warp.ts

```diff
@@ -1,8 +1,11 @@
+import { Tile } from "./tile";
+import { WarpInfo } from "./warp_info";
+
 export class CollisionWarp {
-	public is_collision: boolean;
-	public destination: number;
-	constructor(is_collision?: boolean, destination?: number) {
-		this.is_collision = is_collision;
-		this.destination = destination;
+	public warp_info: WarpInfo;
```

Comment:
Je ne récupère plus seulement la destination mais aussi le type de warp, maintenant qu'ils ont des types (classic => bords de terrain menant à une map adjacente ;  local => d'un warp à un autre sur la même map (why not) ; tp => d'une map à une autre, mais pas forcément en bord, et pas forcément une map adjacente)

Reply:
Pour l'ajout de tile, je l'avais ajouté avant parce que je m'en servais, mais plus maintenant. J'ai hésité à le virer pour le moment, on verra